### PR TITLE
Refer to manual in README

### PR DESCRIPTION
**Pull request by lordofhyphens**
Once the CLI changes are updated and changed, I believe it is preferable to link to an authoritative source of information (the Manual), instead of creating a duplicate that needs to be maintained separately. 

This is an application of the DRY principle (Don't Repeat Yourself).

To be merged after https://github.com/slic3r/Slic3r-Manual/pull/60 is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,301 +88,4 @@ Contributions by Henrik Brix Andersen, Vojtech Bubnik, Nicolas Dandrimont, Mark
...
-                            --ooze-prevention   Drop temperature and park extruders outside a full skirt for automatic wiping
-                            (default: no)
-        --ooze-prevention   Drop temperature and park extruders outside a full skirt for automatic wiping
-                            (default: no)
-        --standby-temperature-delta
-                            Temperature difference to be applied when an extruder is not active and
-                            --ooze-prevention is enabled (default: -5)
-
-For more information about command line usage see the relevant [manual page](http://manual.slic3r.org/advanced/command-line).
+The command line is documented in the relevent [manual page](http://manual.slic3r.org/advanced/command-line).
```
Comment (Oekn5w) [inline README.md]:
```suggestion
The command line is documented in the relevant [manual page](http://manual.slic3r.org/advanced/command-line).
```
Typo

**Comment by Oekn5w** [comment]
Typo in the added line ("relevant")

File: README.md (hunk 1 above)
Comment (alranel) [inline README.md]:
```suggestion
The command line is documented in the relevant [manual page](https://manual.slic3r.org/advanced/command-line).
```

https

**Comment by alranel** [comment]
Totally agree with this

**Pull request merged**